sdk/documenttranslator/ai-document-translator-rest/review/ai-document-translator.api.md

```diff
@@ -0,0 +1,745 @@
+export interface BatchStatusDetail {
+    createdDateTimeUtc: Date;
+    error?: ErrorV2;
+    id: string;
+    lastActionDateTimeUtc: Date;
+    status: Status;
+    // (undocumented)
+    summary: StatusSummary;
+}
+
+// @public (undocumented)
+export interface BatchStatusResponse {
+    nextLink?: string;
+    value: BatchStatusDetail[];
+}
+
+// @public (undocumented)
+export interface BatchSubmissionRequest {
+    inputs: BatchRequest[];
+}
+
+// @public (undocumented)
+export interface CancelOperation {
+    delete(options?: CancelOperationParameters): Promise<CancelOperation200Response | CancelOperation401Response | CancelOperation404Response | CancelOperation429Response | CancelOperation500Response | CancelOperation503Response>;
+    get(options?: GetOperationStatusParameters): Promise<GetOperationStatus200Response | GetOperationStatus401Response | GetOperationStatus404Response | GetOperationStatus429Response | GetOperationStatus500Response | GetOperationStatus503Response>;
+}
+
+// @public
+export interface CancelOperation200Response extends HttpResponse {
+    // (undocumented)
+    body: BatchStatusDetail;
+    // (undocumented)
+    status: "200";
+}
+
+// @public
+export interface CancelOperation401Response extends HttpResponse {
+    // (undocumented)
+    body: ErrorResponseV2;
+    // (undocumented)
+    status: "401";
+}
+
+// @public
+export interface CancelOperation404Response extends HttpResponse {
+    // (undocumented)
+    body: ErrorResponseV2;
+    // (undocumented)
+    status: "404";
+}
+
+// @public
+export interface CancelOperation429Response extends HttpResponse {
+    // (undocumented)
+    body: ErrorResponseV2;
+    // (undocumented)
+    status: "429";
+}
+
+// @public
+export interface CancelOperation500Response extends HttpResponse {
+    // (undocumented)
+    body: ErrorResponseV2;
+    // (undocumented)
+    status: "500";
+}
+
+// @public
+export interface CancelOperation503Response extends HttpResponse {
+    // (undocumented)
+    body: ErrorResponseV2;
+    // (undocumented)
+    status: "503";
+}
+
+// @public (undocumented)
+export type CancelOperationParameters = RequestParameters;
+
+// @public (undocumented)
+export interface DocumentFilter {
+    prefix?: string;
+    suffix?: string;
+}
+
+// @public (undocumented)
+export interface DocumentStatusDetail {
+    characterCharged?: number;
+    createdDateTimeUtc: Date;
+    error?: ErrorV2;
+    id: string;
+    lastActionDateTimeUtc: Date;
+    path?: string;
+    progress: number;
+    sourcePath: string;
+    status: Status;
+    to: string;
+}
+
+// @public (undocumented)
+export interface DocumentStatusResponse {
+    nextLink?: string;
+    value: DocumentStatusDetail[];
+}
+
+// @public (undocumented)
+export type DocumentTranslator = Client & {
+    path: Routes;
+};
+
+// @public (undocumented)
+function DocumentTranslator_2(endpoint: string, credentials: TokenCredential | KeyCredential, options?: ClientOptions): DocumentTranslator;
```

Comment:
_2 sticks out to me. Is this an AE artifact?

Reply:
I think there was a duplicate identifier, I have updated this